mod/editpost.php

```diff
@@ -21,12 +21,18 @@ function editpost_content(App $a)
 	}
 
 	$post_id = (($a->argc > 1) ? intval($a->argv[1]) : 0);
+	$return_url = (($a->argc > 2) ? base64_decode($a->argv[2]) : '');
 
 	if (!$post_id) {
 		notice(L10n::t('Item not found') . EOL);
 		return;
 	}
 
+	// Fallback to SESSION return_path
+	if (empty($return_url)) {
+		$return_url = $_SESSION['return_path'];
```

Comment:
Isn't this supposed to be `$_SESSION['return_url']`?

Reply:
It is! Sorry for that..

Reply:
No problem, you can add a new commit in your branch and submit a new PR.